### ISSUE TITLE
Cannot build the Example app on iOS

### ISSUE THREAD

**Issue by objectiveSee** [react-native-kit/react-native-track-player#762]
# Configuration

```
  React Native Environment Info:
    System:
      OS: macOS 10.15
      CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
      Memory: 1.48 GB / 16.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.14.1 - ~/.nvm/versions/node/v10.14.1/bin/node
      Yarn: 1.3.2 - /usr/local/bin/yarn
      npm: 6.4.1 - ~/.nvm/versions/node/v10.14.1/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 13.1, DriverKit 19.0, macOS 10.15, tvOS 13.0, watchOS 6.0
    IDEs:
      Xcode: 11.1/11A1027 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3
      react-native: 0.59.8 => 0.59.8

```
What `react-native-track-player` version are you using? `latest master`

# Issue
I cannot get the example app to build. In Xcode I get:

```
error: Build input files cannot be found: '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/NowPlayingInfoController/NowPlayingInfoKeyValue.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/TimeEventFrequency.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/Observer/AVPlayerTimeObserver.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/AVPlayerWrapper/AVPlayerWrapper.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/NowPlayingInfoController/NowPlayingInfoProperty.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/NowPlayingInfoController/NowPlayingInfoCenter.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/Event.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/AVPlayerWrapper/AVPlayerWrapperProtocol.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/AVPlayerWrapper/AVPlayerWrapperDelegate.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/RemoteCommandController/RemoteCommand.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/NowPlayingInfoController/MediaItemProperty.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/AudioSessionController/AudioSessionController.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/Observer/AVPlayerObserver.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/RemoteCommandController/RemoteCommandController.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/AudioItem.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/QueueManager.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/AudioSessionController/AudioSession.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/QueuedAudioPlayer.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/APError.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/Observer/AVPlayerItemObserver.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/NowPlayingInfoController/NowPlayingInfoControllerProtocol.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/Observer/AVPlayerItemNotificationObserver.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/NowPlayingInfoController/NowPlayingInfoController.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/AVPlayerWrapper/AVPlayerWrapperState.swift', '/Users/user/Downloads/react-native-track-player-dev/ios/RNTrackPlayer/Vendor/AudioPlayer/SwiftAudio/Classes/AudioPlayer.swift' (in target 'RNTrackPlayer' from project 'RNTrackPlayer')
```

```
The following commands produced analyzer issues:
	Analyze /Users/user/Downloads/react-native-track-player-dev/example/node_modules/react-native/ReactCommon/jsi/jsi.cpp normal x86_64
	Analyze /Users/user/Downloads/react-native-track-player-dev/example/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp normal x86_64
	Analyze /Users/user/Downloads/react-native-track-player-dev/example/node_modules/react-native/React/Base/RCTModuleMethod.mm normal x86_64
(3 commands with analyzer issues)

The following build commands failed:
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(1 failure)
. Run CLI with --verbose flag for more details.
```
# Code

Show us the code you are using

I am using the Example project from the Repo. Running `npm install` then `npm run-ios`. It looks like there are missing Swift files as well as the `RNTrackPlayer` directory

**Comment by objectiveSee** [comment]
I suspect Git submodules to be the issue. I tried to run `git submodule update --init --recursive` and then copy `ios/RNTrackPlayer` into the `example/ios` directory however there are still missing files within `ios/RNTrackPlayer` despite cloning that submodule recently.

**Comment by curiousdustin** [comment]
possibly #703

**Comment by curiousdustin** [comment]
This should be resolved in `1.2.3`
